buildSrc/src/main/kotlin/com/datadog/gradle/utils/MiscUtils.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.utils
+
+import com.android.build.gradle.internal.dsl.ProductFlavor
+import com.google.gson.JsonObject
+import com.google.gson.JsonParser
+import java.io.File
+
+fun json(filePath: String): JsonObject {
```

Comment:
I'd rename this file as `FlavorBuildConfigField.kt`. 

`Utils` is a generic name that doesn't provide info, and `Misc` is even more generic.